### ISSUE TITLE
Incorrect `insecureCapabilities` warning when dropping `all` instead of `ALL`

### ISSUE THREAD

**Issue by daaain** [FairwindsOps/polaris#617]
### Steps to Reproduce

Create a `test.yaml` file with the following contents:

```yaml
apiVersion: batch/v1beta1
kind: CronJob
metadata:
  name: my-cron-job
spec:
  schedule: "0 0 29 2 *"
  startingDeadlineSeconds: 300
  concurrencyPolicy: Forbid
  selector:
    matchLabels:
      app: my-cron-job
  jobTemplate:
    metadata:
      labels:
        app: my-cron-job
    spec:
      backoffLimit: 1
      template:
        spec:
          restartPolicy: "Never"
          containers:
            - name: my-cron-job
              securityContext:
                allowPrivilegeEscalation: false
                runAsNonRoot: true
                capabilities:
                  drop:
                    - all
              command: ["echo", "yay"]
              resources:
                limits:
                  cpu: 500m
                  memory: 512Mi
                requests:
                  cpu: 150m
                  memory: 350Mi
              image: busybox:stable
              imagePullPolicy: Always

```

Then run it through the latest Polaris CLI:

```sh
cat test.yaml | docker run --rm -i quay.io/fairwinds/polaris:4.0.8 polaris audit --only-show-failed-tests --set-exit-code-on-danger --set-exit-code-below-score 85 --audit-path -
```

### Expected Behavior

To get `notReadOnlyRootFilesystem` flagged:

```json
{
  "PolarisOutputVersion": "1.0",
  "AuditTime": "2021-08-27T16:35:59Z",
  "SourceType": "Path",
  "SourceName": "-",
  "DisplayName": "-",
  "ClusterInfo": {
    "Version": "unknown",
    "Nodes": 0,
    "Pods": 0,
    "Namespaces": 0,
    "Controllers": 1
  },
  "Results": [
    {
      "Name": "my-cron-job",
      "Namespace": "",
      "Kind": "CronJob",
      "Results": {},
      "PodResult": {
        "Name": "",
        "Results": {},
        "ContainerResults": [
          {
            "Name": "my-cron-job",
            "Results": {
              "notReadOnlyRootFilesystem": {
                "ID": "notReadOnlyRootFilesystem",
                "Message": "Filesystem should be read only",
                "Details": null,
                "Success": false,
                "Severity": "warning",
                "Category": "Security"
              }
            }
          }
        ]
      },
      "CreatedTime": "0001-01-01T00:00:00Z"
    }
  ],
  "Score": 96
}
```

### Actual Behavior

Instead I also get `insecureCapabilities` despite dropping all:

```json
{
  "PolarisOutputVersion": "1.0",
  "AuditTime": "2021-08-27T16:35:59Z",
  "SourceType": "Path",
  "SourceName": "-",
  "DisplayName": "-",
  "ClusterInfo": {
    "Version": "unknown",
    "Nodes": 0,
    "Pods": 0,
    "Namespaces": 0,
    "Controllers": 1
  },
  "Results": [
    {
      "Name": "my-cron-job",
      "Namespace": "",
      "Kind": "CronJob",
      "Results": {},
      "PodResult": {
        "Name": "",
        "Results": {},
        "ContainerResults": [
          {
            "Name": "my-cron-job",
            "Results": {
              "insecureCapabilities": {
                "ID": "insecureCapabilities",
                "Message": "Container should not have insecure capabilities",
                "Details": null,
                "Success": false,
                "Severity": "warning",
                "Category": "Security"
              },
              "notReadOnlyRootFilesystem": {
                "ID": "notReadOnlyRootFilesystem",
                "Message": "Filesystem should be read only",
                "Details": null,
                "Success": false,
                "Severity": "warning",
                "Category": "Security"
              }
            }
          }
        ]
      },
      "CreatedTime": "0001-01-01T00:00:00Z"
    }
  ],
  "Score": 93
}
```

### Additional Context

Ooooh, just I as was about to post this, I realised that what does work is `drop: ["ALL"]` or

```
drop:
  - ALL
```

But you can find lots of references to the lowercase one like https://snyk.io/blog/kubernetes-securitycontext-linux-capabilities/ or https://kubesec.io/basics/containers-securitycontext-capabilities-drop-index-all/

### References

Couldn't find any, so I opened this new issue.

**Comment by rbren** [comment]
Thanks for the report! I have a PR to fix this here: https://github.com/FairwindsOps/polaris/pull/619

From what I can tell, it's just `all`/`ALL`, and other capabilities are always uppercase. Still trying to verify this though.

**Comment by rbren** [comment]
After some research, I found that _all_ capabilities are case-insensitive! Modified the PR to make all the checks case-insensitive.

**Comment by rbren** [comment]
Fixed in 4.1
